modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/data/handler/BatchEnginePortletDataHandler.java

```diff
@@ -337,7 +339,7 @@ protected void doPrepareManifestSummary(
 
 		Map<String, Serializable> parameters =
 			BatchEnginePortletDataHandlerUtil.buildExportParameters(
-				Collections.emptyList(), portletDataContext);
+				Collections.emptyList(), null, portletDataContext);
```

Comment:
why don't we pass Collections.emptyList() also for the new param like we are doing for nestedFields?